summernote/summernote
# Fix jQuery dependency check - upper/lowercase issue

#### What does this PR do?

The jQuery object is registered with the name "jQuery" in the current document. The import was therefore changed from jquery to jQuery.
This commit fixed #4539

#### Where should the reviewer start?

settings.js

#### How should this be manually tested?

Build the summernote project and open a website with summernote-editor.
The editor should be visible.

#### Any background context you want to provide?

The jQuery extension registers the object jQuery with uppercase Q.
The dependency check to "jquery" was therefore failing.

#### What are the relevant tickets?
ticket #4539

#### Screenshot (if for frontend)


### Checklist
